code/baselineTiming.py

```diff
@@ -34,49 +34,52 @@ def main():
     # utils.preprocessing_merg(rawMERG)
     # ---------------------------------------------------------------------------------
     # ---------------------------------- user inputs --------------------------------------
-    DIRS['mainDirStr'] = "/Users/kwhitehall/Documents/capstones/usc2015/baselineTimings/testSeparate"#"/directory/to/where/to/store/outputs"
-    DIRS['TRMMdirName'] = "/Users/kwhitehall/Documents/capstones/usc2015/baselineTimings/datadir/TRMM"#"/directory/to/the/TRMM/netCDF/files"
-    DIRS['CEoriDirName'] = "/Users/kwhitehall/Documents/capstones/usc2015/baselineTimings/datadir/MERG"#"/directory/to/the/MERG/netCDF/files"
+#    DIRS['mainDirStr'] = "/Users/youssefbiaz/Documents/USC/,2015-3Fall'15/5CSCI401/grab-tag-graph/baselineTimings/output/baselineTimings"#"/directory/to/where/to/store/outputs"
+    #DIRS['TRMMdirName'] = "/Users/youssefbiaz/Documents/USC/,2015-3Fall'15/5CSCI401/grab-tag-graph/baselineTimings/datadir/TRMM"#"/directory/to/the/TRMM/netCDF/files"
+    #DIRS['CEoriDirName'] = "/Users/youssefbiaz/Documents/USC/,2015-3Fall'15/5CSCI401/grab-tag-graph/baselineTimings/datadir/MERG"#"/directory/to/the/MERG/netCDF/files"
     #get the dates for analysis
-    startDateTime = "200908310000" #"yyyymmddhrmm"
-    endDateTime = "200908312100"
+    #startDateTime = "200908310000" #"yyyymmddhrmm"
+    #endDateTime = "200908312100"
+
+    userVariables = variables.define_user_variables()
+    graphVariables = variables.define_graph_variables()
     # ---------------------------------- end user inputs --------------------------------------
     # Checks that inputs are ok
     try:
-        if not os.path.exists(DIRS['CEoriDirName']):
+        if not os.path.exists(userVariables.DIRS['CEoriDirName']):
             print "Error! MERG invalid path!"
-            DIRS['CEoriDirName'] = raw_input("> Please enter the directory to the MERG netCDF files: \n")
+            userVariables.DIRS['CEoriDirName'] = raw_input("> Please enter the directory to the MERG netCDF files: \n")
     except:
         print "..."
 
     try:
-        if not os.path.exists(DIRS['TRMMdirName']):
+        if not os.path.exists(userVariables.DIRS['TRMMdirName']):
             print "Error: TRMM invalid path!"
-            DIRS['TRMMdirName'] = raw_input("> Please enter the location to the raw TRMM netCDF files: \n")
+            userVariables.DIRS['TRMMdirName'] = raw_input("> Please enter the location to the raw TRMM netCDF files: \n")
     except:
         pass
 
     try:
-        if not os.path.exists(DIRS['CEoriDirName']):
+        if not os.path.exists(userVariables.DIRS['CEoriDirName']):
             print "Error! MERG invalid path!"
-            DIRS['CEoriDirName'] = raw_input("> Please enter the directory to the MERG netCDF files: \n")
+            userVariables.DIRS['CEoriDirName'] = raw_input("> Please enter the directory to the MERG netCDF files: \n")
     except:
         print "..."   
```

Comment:
Hi @youssefbiaz  this looks like a redundant try block for userVariables.DIRS['CEoriDirName']